src/mscorlib/shared/System/IO/Path.Unix.cs

```diff
@@ -189,6 +194,14 @@ public static string GetPathRoot(string path)
             return IsPathRooted(path) ? PathInternal.DirectorySeparatorCharAsString : String.Empty;
         }
 
+        public static ReadOnlySpan<char> GetPathRoot(ReadOnlySpan<char> path)
+        {
+            if (PathInternal.IsEffectivelyEmpty(path))
+                return ReadOnlySpan<char>.Empty;
+
+            return IsPathRooted(path) ? PathInternal.DirectorySeparatorCharAsString.AsReadOnlySpan() : ReadOnlySpan<char>.Empty;
```

Comment:
Could this be simplified to just:
```C#
return !PathInternal.IsEffectivelyEmpty(path) && IsPathRooted(path) ?
    PathInternal.DirectorySeparatorCharAsString.AsReadOnlySpan() :
    ReadOnlySpan<char>.Empty;
```
rather than having two separate conditions that return Empty?

Reply:
should I make it inline ?